### ISSUE TITLE
subscription tab should be updated/can be improved

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#27]
### Issue

I have tried to subscribe to like 6 channels but the interface is not helpful.
check the attached screenshot
![Screenshot_20201105-222304](https://user-images.githubusercontent.com/39449563/98272126-bec4a780-1fb6-11eb-865a-3b3b7fc2bb20.png)

Newpipe has a much better way of handling hundreds of channel subscriptions.

![Screenshot_20201105-223308](https://user-images.githubusercontent.com/39449563/98272432-1531e600-1fb7-11eb-98cb-e5d8a4662824.png)

V. 1.5.2

**Comment by stom79** [comment]
You can scroll horizontally accounts, the idea is to quickly filter videos by clicking on them.

**Comment by test2a** [comment]
@stom79 yes. this works when you have just a few channels. as i wrote above, the above screenshot shows only 3.5 icons. then pressing all does nothing.

okay. if we have to use this interface, how about allowing user to set the favourite "4" pinned channels. rest can be in "all" where the channel icons show up in a grid alphabetically or any other sort as chosen by the user? that should work out nice
